web/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,248 @@
+import Link from "next/link";
+import { SetStateAction, useState } from "react";
+import { MdAccountCircle, MdAddCircle } from "react-icons/md";
+
+import Button from "../../components/Inputs/Button";
+import LinkIconButton from "../../components/Inputs/LinkIconButton";
+import SearchBar from "../../components/Inputs/SearchBar";
+import ImageCard from "../../components/Surfaces/ImageCard";
+
+/*
+ *
+ * Note to viewer:
+ * Please bully, I must learn the error in my ways.
+ *
+ * */
+
+export default function Index() {
+  const [text, setText] = useState("");
+
+  function handleChange(value: string) {
+    setText(value);
+  }
```

Comment:
I would remove this definition and just inline `setText()` below. `handleChange()` isn't necessary because all it does is call another function with exactly the same arguments.

Reply:
I know right now it doesn't do much, but I'm assuming there will be a need to have this once this is linked with the API. 
i.e. Calling text filtering?

I'll make the changes for now though.